column/__init__.py

```diff
@@ -7,10 +7,16 @@
 from subprocess_runner import SubprocessRunner
 from callback import AnsibleCallback
 
+import ConfigParser
+
+DEFAULT_CONF_FILE = '/etc/column/column.conf'
 
 __all__ = [
     'APIRunner', 'Runner', 'SubprocessRunner', 'AnsibleCallback'
 ]
 
 
 __version__ = '0.3.6'
+
+cfg = ConfigParser.ConfigParser(allow_no_value=True)
+cfg.read(DEFAULT_CONF_FILE)
```

Comment:
What happens if this file does not exist? This will be executed globally for column, even when column is used as library and not as a service.